src/helpers/discord_utils.ts

```diff
@@ -120,12 +122,14 @@ export async function sendOptionsMessage(message: Eris.Message<Eris.GuildTextabl
     seekTypeString = updatedOption == GameOption.SEEK_TYPE ? bold(seekTypeString) : codeLine(seekTypeString);
     modeTypeString = updatedOption == GameOption.MODE_TYPE ? bold(modeTypeString) : codeLine(modeTypeString);
     goalString = updatedOption == GameOption.GOAL ? bold(goalString) : codeLine(goalString);
+    timerString = updatedOption == GameOption.TIMER ? bold(timerString) : codeLine(timerString);
 
     let goalMessage = `First one to ${goalString} points wins.`;
+    let timerMessage = ` in less than ${timerString} seconds`;
```

Comment:
Move space to the string

Reply:
Can you think of an elegant way for this one?

Reply:
nevermind, looks fine